src/ResourceManager/Websites/Commands.Websites/Cmdlets/DeploymentSlots/SetAzureWebAppSlot.cs

```diff
@@ -80,8 +80,12 @@ public class SetAzureWebAppSlotCmdlet : WebAppSlotBaseCmdlet
         [ValidateNotNullOrEmpty]
         public bool Use32BitWorkerProcess { get; set; }
 
-        [Parameter(Mandatory = false, HelpMessage = "Destination slot name for auto swap")]        
+        [Parameter(Mandatory = false, HelpMessage = "Destination slot name for auto swap")]
+
```

Comment:
@LukaszStem all your bool properties are not NULLABLE, but during SiteConfig construction they are treated and made nullable, any reason why the datatype is changed?